_posts/2024-09-28-unstable-release-0.10.9.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Stable release 0.10.9
+author: Saugia
+---
+
+If you like more Endless Sky, boy do I have the thing for you! v0.10.9 is now live on Github and Steam! As usual, GoG will be within a few days pending some QA testing. This is a hefty unstable release that brings all sorts of content additions, adjustments, balance changes, bug fixes, and so on! If you're itching to know everything that has changed, head on over to the changelog for the release, which can be found here:
+
+Here are some highlights coming with this release:
+
+When it comes to boarding and capturing, there never really has been a reason to incorporate defense as the game always calculates using the attack values of the AI versus your crew. With this update, if both sides choose to attack, two rounds of combat are run - one with the player's attack against the enemy's defense, and one with the enemy's attack against the player's defense, instead of using the player's attack against the enemy's attack. Now those thousands of Pug Biodefenses you farmed can be useful against any trigger-happy Pirate! Thanks to the efforts of Peter and Hurleveur. But that's not all that these two have for us!
+
+Have you ever wanted your escorts to fly with you more uniformly? To enter a system in style, or to enter one with a mroe strategic lineup? Well, now you can, with the addition of Fleet Formations! Using Shift + your "fleet gather" key (G by default), you can now cycle through several fleet formations, such as circles, lines, daggers, and several other shapes! Once again thanks to Peter.
+
+<img class="centered shadowed" src="/images/blog/v0.10.9/formation.png" width="450" height="498" />
+
+And if that isn't enough for you fleet-wise, we now have the long-requested Hold Fire command, which you can find in the Controls tab of the game's settings if you wish to change it to a different key. And once again, thanks to Hurleveur.
+
+For the more meaty part of this update, we have not one, but once again, TWO new species that have appeared in the Endless Sky galaxy, each with their own territory to explore and a few interactions to be had. Be on the lookout for the Incipias authored by @risingleaf, and the Successors from @Daerii! That's all I would like to say about them, however, if you want to peek over back to Gegno space, you might find a few new missions there too.
```

Comment:
I assume these are Discord usernames?

Reply:
They're GitHub usernames now.

Reply:
How do you mean?

I'm thinking that if they're the usernames the people is question use on Discord, it's probably not a problem, but otherwise it should be changed to RisingLeaf and Daeridanii1.